docs/developers/prerequisites.md

```diff
@@ -0,0 +1 @@
+# Prerequisites
```

Comment:
@dakotabenjamin My thinking is that maybe this is not required as a separate section but we make sure any prerequisites are included in the dev setup section above. We can discuss on Tuesday :)